src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectOndertekenenService.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.informatieobjecten;
+
+import com.fasterxml.uuid.impl.UUIDUtil;
+
+import net.atos.client.zgw.drc.DRCClientService;
+import net.atos.client.zgw.drc.model.EnkelvoudigInformatieobjectWithInhoudAndLock;
+import net.atos.client.zgw.drc.model.InformatieobjectStatus;
+import net.atos.client.zgw.drc.model.Ondertekening;
+import net.atos.client.zgw.drc.model.OndertekeningSoort;
+import net.atos.zac.authentication.LoggedInUser;
+import net.atos.zac.enkelvoudiginformatieobject.EnkelvoudigInformatieObjectLockService;
+import net.atos.zac.enkelvoudiginformatieobject.model.EnkelvoudigInformatieObjectLock;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+
+@ApplicationScoped
+@Transactional
+public class InformatieObjectOndertekenenService {
+
+    @Inject
+    private Instance<LoggedInUser> loggedInUserInstance;
+
+    @Inject
+    private EnkelvoudigInformatieObjectLockService enkelvoudigInformatieObjectLockService;
+
+    @Inject
+    private DRCClientService drcClientService;
+
+    public void ondertekenen(final List<UUID> enkelvoudigInformatieObjectUUIDs) {
```

Comment:
Noem de methoden naar wat ze doen. Iets als ondertekenEnkelvoudigInformatieObjecten en ondertekenEnkelvoudigInformatieObject. Gebruik Onderteken ipv ondertekenen.

Reply:
Hernoemd in nieuwe commit